app/build.gradle

```diff
@@ -44,12 +44,13 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
         useIR = true
+        freeCompilerArgs += "-Xallow-unstable-dependencies"
```

Comment:
It would be nice to add todo here, so it is removed while koin supports compose beta